packages/sharec-core/lib/steps/readConfigs.js

```diff
@@ -18,12 +19,34 @@ const readConfigs = async (input) => {
   try {
     const configsPaths = await find(configsPath, '**/*')
     const withoutLocks = configsPaths.filter((config) => !/(\.|-)lock/.test(config))
+
+    const withoutBinaries = []
```

Comment:
Better to add `readBinaries` step instead of extending current one

Reply:
Adding a `readBinaries` step would require to duplicate the test `isText(file) || getEncoding(await readBuffer(file, 24)) === 'utf8'` which can be expensive if the file need to be read.

Reply:
I've replaced it with a `copyBinaries`.